google/cloud/forseti/services/server.py

```diff
@@ -75,6 +80,15 @@ def serve(endpoint,
     if enable_console_log:
         logger.enable_console_log()
 
+    if enable_profiler:
```

Comment:
Hmmm, I think that it would be nicer to start the profiler in the `main()` function.  #1, it keeps the serve() cleaner here and one less parameter to pass & specify, and #2 we can trace other things in `main()` in case they get added in the future. I recommend moving this to right before `serve()` is called in the `main()`